packages/infra/client/actor-kv/src/metadata.rs

```diff
@@ -0,0 +1,9 @@
+use serde::Serialize;
+
+#[derive(Clone, PartialEq, ::prost::Message, Serialize)]
+pub struct Metadata {
+	#[prost(string, tag = "1")]
+	pub kv_version: ::prost::alloc::string::String,
```

Comment:
can we make this a raw buffer so we don't have the overhead of strings?

disregard if changed upstack